astrogrism/config/HST/create_HST_reffiles.py

```diff
@@ -0,0 +1,19 @@
+from pathlib import Path
+
+from astrogrism.config.HST.reference_file_generators._generate_specwcs import create_grism_specwcs  # noqa
+from astrogrism.config.HST.reference_file_generators._generate_wavelengthrange import create_tsgrism_wavelengthrange  # noqa
+# from astrogrism.config.HST.reference_file_generators.reference_file_generators._generate_distortion import create_distortion  # noqa
+
+
+def create_reference_files(conffile, hst_grism, outpath=Path.cwd(), outbasename=None):
+    if not outbasename:
```

Comment:
```suggestion
    if outbasename is None:
```
I prefer checking for `None` explicitly if that's what we're looking for, rather than seeing if it evaluates to False.

Reply:
Sure, no strong opinions here!